telemetry/specs/INT.mdk

```diff
@@ -384,45 +384,95 @@ Considering all these factors, the INT specification requires that an INT
 switch **must not fragment** packets in order to append INT information to
 the packet.
 
-### Checksum Update
+## Checksum Update
 
 INT may be transported over an L4 protocol such as TCP or UDP, or over an
-encapsulation header that includes an L4 header, such as VXLAN. In these cases,
-an INT source or transit switch may be required to update the L4 Checksum of
-the encapsulating header.
-
-In some cases, an L4 Checksum update is not necessary. For example, when UDP is 
-transported over IPv4, it is possible to assign a zero Checksum, causing the 
-receiver to ignore the value of the Checksum field. For UDP over IPv6, there
-are specific use cases in which it is possible to assign a zero Checksum 
-(as defined in RFC 6936).
-
-If an L4 Checksum update is required, an INT source/transit switch may perform
-it in one of two ways:
-
-* Update the L4 Checksum field such that the new value is equal to the Checksum 
+encapsulation header that includes an L4 header, such as VXLAN.
+
+In theory, since each INT transit hop modifies the Layer 4 payload, it should
+update the Layer 4 checksum. Layer 4 checksum update at transit hops is
+necessary to protect the integrity of INT metadata between the INT source
+and INT sink. If integrity of INT metadata is to be protected by Layer 4
+checksum, each INT transit hop must perform incremental checksum calculation
+as complete checksum calculation at a transit hop will mask any corruption
+in the INT metadata inserted by previous INT hops.
```

Comment:
Since this paragraph is so early in this section, it reads as if protecting the integrity of INT metadata is the key concern when determining whether to maintain checksum integrity when adding/removing INT metadata. I don't believe that is the case. If this justification is to remain, it should be discussed after the UDP checksum zero and encapsulation protocol aspects, and integrity of the L4 payload.